client/me/help/help-contact/index.jsx

```diff
@@ -204,6 +207,26 @@ module.exports = React.createClass( {
 
 	onOperatorsAway: function() {
 		this.showOperatorAvailabilityNotice( false );
+
+		//Autofill the subject field since we will be showing it now that operators have went away.
+		this.autofillSubject();
+	},
+
+	/**
+	 * Auto fill the subject with the first five words contained in the message field of the contact form.
+	 */
+	autofillSubject: function() {
+		const contactForm = this.state.contactForm;
+
+		if ( ! contactForm.message || contactForm.subject ) {
+			return;
+		}
+
+		const words = contactForm.message.split( /\s+/ );
+
+		contactForm.subject = words.slice( 0, 5 ).join( ' ' ) + '…';
```

Comment:
@dmsnell I modified this so that it no longer sets state. I changed it because the state was changing unnecessarily and causing a bunch of renders. Now I'm just calling `forceUpdate` when a render is needed.